mycompany-seeq-link-connector-myconnector/build.gradle.kts

```diff
@@ -6,14 +6,17 @@ plugins {
 group = "com.mycompany.seeq.link.connector"
 version = "0.1.0-SNAPSHOT"
 
+project.version = "1.0.0.0"
+val minimumSeeqLinkSdkVersion = "100.3.349"
+
 dependencies {
-    compileOnly("com.seeq.link:seeq-link-sdk:66.0.0-v202410141803")
+    compileOnly("com.seeq.link:seeq-link-sdk:100.3.349")
```

Comment:
How hard do you think it would be to extract this version into one single place? (not necessary in this PR but I think it would be great if we can do this easily).
Maybe we can have it in gradle settings or at least extract it in a variable here